inst/text/stylecss.txt

```diff
@@ -0,0 +1,41 @@
+/* Setting header colour */
+  h1, .h1, h2, .h2, h3, .h3, h4, .h4, h5, .h5, h6, .h6 {
+    margin-top: 20px;
+    margin-bottom: 10px;
+    color: #3F3685;
```

Comment:
In the 'shiny CSS file' there's some header code which sets up the various colours as variables. I wasn't able to get that to work here, which is why I just hardcoded the colours like this.

I don't know CSS but it would be much better to have these as named PHS colours if possible.

Reply:
Do you have an RMarkdown file I can do some testing with?

Reply:
https://phs-git.nhsnss.scot.nhs.uk/ATI/Git-Training This is what I was inspired by!

Reply:
I've added in the colour definitions to the top like in the shiny css file - it all seems to work when I tested